capitoli/Terzo_Passo_Variabili.rst

```diff
@@ -35,6 +38,7 @@ SCATOLA1 = 7
 il simbolo :boltred:`=` significa:     nella scatola di nome SCATOLA1 metti il numero 7.   
 
 *DOMANDA*
```

Comment:
Non è chiaro lo spazio, non si capisce.
Metterei un'altra riga vuota sopra e poi il testo più grosso oppure in grassetto